src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1266,3 +1266,29 @@ ingress-nginx:
 ```
 
 The full list of values is [available here](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml).
+
+#### Okteto ingress-nginx default values
+
+Okteto Self Hosted applies some defaults to ingress-nginx, which can be checked before overriding them with the following command:
+
+```
+helm get values okteto/okteto --jsonpath '{.ingress-nginx}'
+```
+
+#### Use a different ingressClass
+
+Okteto Self Hosted uses the following ingressClass value by default in both, control plane and dev user ingresses: `okteto-nginx`.
```

Comment:
What do we mean by "control plane" and "dev user" ingresses? Do we use those terms anywhere else in the docs?

```suggestion
Okteto Self Hosted uses the following ingressClass value by default for control plane and dev user ingresses: `okteto-nginx`.
```